### DISCUSSION TITLE
Auto Focus PR name input box when "edit" is clicked

### DISCUSSION THREAD

**Discussion by davidcoleman007** [community/community#112497] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When i click the "edit" button on a pr (seen below)
![image](https://github.com/community/community/assets/41075/059a0ddd-b937-4408-807b-7a4890e335d0)

I expect that i will be automatically focused on the text entry area.

I also expect that if I can hit "G P" to go to pull requests page, and control f to find a pr, and then esc to close my find, and enter to open the pr, then i should have a kb shortcut to edit this pr name and be automatically focused on the entry box.

**Comment by kasipavankumar** [comment]
+1

Auto focus works when editing issues title. It'd be nice to have the same auto focus behaviour for when editing the PR titles as well.

Issue title editing

https://github.com/user-attachments/assets/a9d3d399-445c-46ec-ad66-8ef04d94633a

Pull Request title editing

https://github.com/user-attachments/assets/815ab865-3259-45f5-9930-09c01d8d529e
